modules/measure/src/main/java/com/opengamma/strata/measure/swap/SwapMeasureCalculations.java

```diff
@@ -58,9 +59,14 @@
 final class SwapMeasureCalculations {
```

Comment:
Just to note that the XMeasureCalculations classes vary between final and non-final (although this only affects us due to their scope) - probably worth standardising these in a future patch.